ilc/common/router/ServerRouter.js

```diff
@@ -43,6 +41,40 @@ module.exports = class ServerRouter {
         }
     }
 
+    async getRouteAssets(reqUrl) {
```

Comment:
I think it's better to have this code in a form of private method inside `configs-injector.js`
It has little in common with routing

Reply:
Moved